api/db/mongoose.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+
+const mongo_uri = 'mongodb://localhost/hyf';
+mongoose.connect(mongo_uri, { useNewUrlParser: true }, function(err) {
+  if (err) {
+    throw err;
+  } else {
+    console.log(`Successfully connected to ${mongo_uri}`);
+  }
+});
```

Comment:
```suggestion
const mongoose = require('mongoose');
const { MONGO_CONNECTION_URI } = require('./config.js');

const mongoUri = MONGO_CONNECTION_URI;
mongoose.connect(mongoUri, { useNewUrlParser: true }, function(err) {
  if (err) {
    throw err;
  } else {
    console.log(`Successfully connected to ${mongoUri}`);
  }
});
```

Reply:
Hey @Toinne . I tried to do this after I sent this PR but it break the application on the way that I did. It was because I put the mongo link on the .env, instead of `localhost/api` or so. Thanks to point it.